public-html/addbook.php

```diff
@@ -1,3 +1,14 @@
+<?php session_start();
+include "../resources/src/books.php";
```

Comment:
@anjali-rathod i am assume you are still working on this PR, but make sure to remove this include statement for books.php since you are now using autoloader hence include that here.

Reply:
Yes, @guptahemant . @shriaas2898  will do it once my code is merged